proto-build/src/main.rs

```diff
@@ -361,9 +298,6 @@ fn copy_and_patch(src: impl AsRef<Path>, dest: impl AsRef<Path>) -> io::Result<(
             "/// Generated server implementations.\n\
              #[cfg(feature = \"grpc\")]",
         ),
-        // Use `tendermint_proto` as source of `google.protobuf` types
-        // TODO(tarcieri): figure out what's wrong with our `buf` config and do it there
-        ("::prost_types::", "::tendermint_proto::google::protobuf::"),
```

Comment:
Oh hey, thanks for cleaning this up, I forgot to remove it when we fixed the `buf` config